src/lib.rs

```diff
@@ -123,6 +123,9 @@ mod init;
 pub mod pbkdf2;
 
 pub mod rand;
+mod limb;
+pub use limb::GFp_rand_mod;
```

Comment:
It already looks like this:

```Rust
#[allow(unsafe_code)]
#[allow(non_snake_case)]
#[doc(hidden)]
#[no_mangle]
pub unsafe extern fn GFp_rand_mod ...
```

But that's not enough, I get an error if I delete the `pub use`, I think because the `limb` module is private.